src/applications/facility-locator/constants/index.js

```diff
@@ -58,7 +58,7 @@ export const LOCATION_OPTIONS = [
 /**
  * Defines the ± change in bounding box size for the map when changing zoom
  */
-export const BOUNDING_RADIUS = 0.75;
+export const BOUNDING_RADIUS = 1.4;
```

Comment:
Just out of curiosity, was there any specific reason for increasing the radius this much?

Reply:
Just seemed like the sweet spot for results returned - not too many, not too few.